data-raw/cancovid.R

```diff
@@ -1,25 +0,0 @@
-## code to prepare `cancovid` dataset goes here
-
-path_to_csvs <- here::here("../../COVID-BC/Covid19Canada/updates.nosync/")
-files <- list.files(path_to_csvs)
-ca_as_ofs <- as.Date(substr(files, 1, 10)) %>%
-  intersect(fc_time_values) %>%
-  as.Date(origin = "1970-01-01")
-
-can <- purrr::map(ca_as_ofs, ~ {
-  readr::read_csv(here::here(path_to_csvs, paste0(.x, ".csv"))) %>%
-    left_join(ca_pop) %>%
-    mutate(time_value = lubridate::dmy(date_report)) %>%
-    filter(province %in% ca_pop$province, time_value > "2020-04-01") %>%
-    mutate(geo_value = province,
-           case_rate = cases / population * 1e5) %>%
-    select(geo_value, time_value, case_rate) %>%
-    as_epi_df(geo_type = "province", as_of = .x)
-})
-names(can) <- ca_as_ofs
-can <- can %>% bind_rows(.id = "version") %>%
-  mutate(version = lubridate::ymd(version))
-saveRDS(can, "inst/extdata/can_prov_cases.rds")
-
-
-usethis::use_data(cancovid, overwrite = TRUE)
```

Comment:
> I've dropped the cancovid.R script since it depends on non-included raw data files, and is not exported/documented by the package.

@dajmcdon was this supposed to still exist in this package?

Reply:
Yes. It's used here: https://github.com/cmu-delphi/epipredict/blob/8d7c5e5bdf109334599211de9e0f273fb42edb11/vignettes/articles/sliding.Rmd#L194

I think I haven't gotten to fixing that part up because it's painful, and I wasn't sure what I wanted to do (or even if I could provide code to recreate at this point). It involved downloading the GitHub repo from https://github.com/ccodwg/CovidTimelineCanada and running a script to back out all the versions. Then running that R script on the csvs. Perhaps @nmdefries or someone else can try to fix it up?

Reply:
Actually, here's the fork with the script that does it (though out of date now). https://github.com/zcaiElvis/Covid19Canada

Reply:
Thanks, Dan, will take a look!

Reply:
Added the existing `cancovid.R` back and added a stub documenting and exporting the `cancovid` dataset. Will fix the script up in a separate PR.